experiments/dataSources/openAI/ChatGPT_Demo.xml

```diff
@@ -0,0 +1,265 @@
+<skuid__page unsavedchangeswarning="yes" personalizationmode="server" showsidebar="false" showheader="false" theme="ChatGPT_Demo">
+	<models>
+		<model id="Create_Chat_Completion" query="false" datasource="OpenAI" payloadcontenttype="JSON" sendchanges="usetemplaterequest" customrequesttemplate="{&#10;    &quot;model&quot;: &quot;gpt-3.5-turbo-0301&quot;,&#10;    &quot;messages&quot;: [ {{#$Model.Chat_Thread.data}}{{comma}}{&quot;content&quot; : {{stringifiedContent}}, &quot;role&quot; : &quot;{{role}}&quot;}{{/$Model.Chat_Thread.data}} ]&#10;}" contenttype="application/json" createrowifnonefound="false" verb="POST" endpoint="v1/chat/completions" payloadsnippet="(not in use) requestBody" useresponsesnippet="false" responsesnippet="responseHandler">
```

Comment:
The endpoint in this model has "v1" while the general instructions for setting up the data source in this experiement also has "v1"  -  so the network call fails because it has "v1/v1"....